llvm-external-projects/iree-dialects/include/iree-dialects/Transforms/TransformMatchers.h

```diff
@@ -136,13 +136,13 @@ struct IsProjected : public SingleValuePredicateParam<int64_t> {
 struct IsIdentity {};
 
 /// Predicate tag indicating that the operand is a special float constant.
-struct ConstantFloatMin {};
+struct ConstantFloatMinOrMinusInf {};
 struct ConstantFloatZero {};
 
 /// Predicate indicating that the operand is the same value as its producer's
 /// operand.
 struct SameOperandAsProducer
-    : public SingleValuePredicateParam<std::pair<int64_t, int64_t>> {
+    : public SingleValuePredicateParam<SmallVector<int64_t>> {
```

Comment:
Could you document the semantics of values in this vector?

Reply:
Note that #12168 removes this completely in favor of value matchers.

Reply:
awesome, thanks Alex